grammars/clojure.cson

```diff
@@ -279,25 +279,24 @@
     'patterns': [
       {
         # ns, declare and everything that starts with def* or namespace/def*
-        'begin': '(?<=\\()(ns|declare|def[\\w\\d._:+=><!?*-]*|[\\w._:+=><!?*-][\\w\\d._:+=><!?*-]*/def[\\w\\d._:+=><!?*-]*)\\s+'
+        'begin': '(?<=\\()(ns|declare|def[0-9#\'A-Z_a-z!$%&+\\-.:<=>?|*]*|[A-Z_a-z!$%&+\\-.:<=>?|*][0-9#\'A-Z_a-z!$%&+\\-.:<=>?|*]*/def[0-9#\'A-Z_a-z!$%&+\\-.:<=>?|*]*)\\s+'
         'beginCaptures':
           '1':
             'name': 'keyword.control.clojure'
         'end': '(?=\\))'
-        'name': 'meta.definition.global.clojure'
```

Comment:
I think this `meta` should be kept and the actual function changed to just `entity.name.global.clojure`.